exec/java-exec/src/main/java/org/apache/drill/exec/rpc/data/DataConnectionCreator.java

```diff
@@ -66,8 +67,8 @@ public DataTunnel getTunnel(DrillbitEndpoint endpoint) {
     return new DataTunnel(newManager);
   }
 
-  public void close() {
-    Closeables.closeQuietly(server);
+  public void close() throws IOException {
```

Comment:
You can use `AutoCloseables.close(server, logger)` which catch any exception and log it as a warning in the passed logger